region/maridia/inner-yellow/Plasma Room.json

```diff
@@ -388,6 +389,61 @@
       "requires": [],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 3],
```

Comment:
I think I would rather this go to node 2. Took me a bit to decide, really, but here my weak reasons for why:
- It will follow the room diagram (though that could be edited)
- It technically won't kill the top pirate (though you could do that on room exit)
- There's no [3, 2] link, so this wouldn't let you also collect the item (though it could be added)

Honestly, node 3 should probably be deleted, but if you want to keep this as [1, 3], let me know